docs/source/conf.py

```diff
@@ -44,6 +44,10 @@
 
 master_doc = "index"
 
+# Configure smartquotes to only transform quotes and ellipses, not dashes
+smartquotes_action = "qe"
```

Comment:
As an alternative to this we can just switch off smartquotes entirely; that would match the manpages and podman-remote windows HTML. I've gone for this because the smart quotation marks look nicer and don't cause any errors as far as I can see.